zcash_client_sqlite/src/wallet/orchard.rs

```diff
@@ -597,6 +597,7 @@ pub(crate) mod tests {
 
     #[test]
     #[ignore] // FIXME: #1316 This requires support for dust outputs.
+    #[cfg(not(feature = "expensive-tests"))]
```

Comment:
Why are these `cfg(not(...))` instead of just `cfg(...)`?

Reply:
So, it's annoying. CI uses the -- --ignored flag to run _just_ the ignored tests. So I want this test to _actually_ be ignored, so I made the slow tests CI workflow use the `expensive-tests` feature, and then I opt this out of the slow tester by adding this cfg attr.